integration_tests/loop_10.py

```diff
@@ -49,9 +49,68 @@ def nested_loop_for_for():
         return
     assert False
 
+def nested_loop_for_while():
+    i: i32
+    j: i32 = 10
+    for i in range(2):
+        print("outer: " + str(i))
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_while_for():
+    i: i32 = 0
+    j: i32
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        for j in range(10, 20):
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_while_while():
+    i: i32 = 0
+    j: i32 = 10
+    while i < 2:
+        print("outer: " + str(i))
+        i += 1
+        while j < 20:
+            print("  inner: " + str(j))
+            break
+    else:
+        print("no break in outer loop")
+        return
+    assert False
+
+def nested_loop_else():
+    i: i32
+    j: i32
+    for i in range(2):
+        print("outer: " + str(i))
+        for j in range(10, 12):
+            print("  inner: " + str(j))
+        else:
+            print("no break in inner loop")
+    else:
+        print("no break in outer loop")
+        return
+    assert False
```

Comment:
Its good that we are adding tests. I think I found a bug in the `for-else` implementation.
```console
% git diff
diff --git a/integration_tests/loop_10.py b/integration_tests/loop_10.py
index 224c0e21d..c40d2863b 100644
--- a/integration_tests/loop_10.py
+++ b/integration_tests/loop_10.py
@@ -105,12 +105,12 @@ def nested_loop_else():
     assert False
 
 
-with_break_for()
-with_break_while()
-no_break_for()
-break_in_if_for()
-nested_loop_for_for()
-nested_loop_for_while()
-nested_loop_while_for()
-nested_loop_while_while()
+# with_break_for()
+# with_break_while()
+# no_break_for()
+# break_in_if_for()
+# nested_loop_for_for()
+# nested_loop_for_while()
+# nested_loop_while_for()
+# nested_loop_while_while()
 nested_loop_else()
% python integration_tests/loop_10.py
outer: 0
  inner: 10
  inner: 11
no break in inner loop
outer: 1
  inner: 10
  inner: 11
no break in inner loop
no break in outer loop
% lpython integration_tests/loop_10.py
outer: 0
  inner: 10
  inner: 11
outer: 1
  inner: 10
  inner: 11
no break in outer loop
```